graphql-dgs-codegen-core/src/main/kotlin/com/netflix/graphql/dgs/codegen/CodeGenCli.kt

```diff
@@ -46,7 +46,7 @@ class CodeGenCli : CliktCommand("Generate Java sources for SCHEMA file(s)") {
     )
     private val language by option("--language", "-l", help = "Output language").choice("java", "kotlin", ignoreCase = true)
         .default("java")
```

Comment:
Curious about why you need to only update the CLI here. We've used the CLI mainly for tests, but as a user, it's the generateJava task that's part of the plugin that needs the config option. In any case, you'll want to update that in the plugin core with this option.

Reply:
Updated generateJava as well with the feature